CMakeLists.txt

```diff
@@ -57,7 +57,7 @@ add_subdirectory(${CMAKE_CURRENT_SOURCE_DIR}/source)
 # TESTS
 ########################################
 
-option(ERA_GP_SIM_TESTS "Enable tests" OFF)
+option(ERA_GP_SIM_TESTS "Enable tests" ON)
```

Comment:
Eh, no. Unless you remember to turn this off in a few months :blush: The idea is that users who want to build our project without tests don't have to compile the tests. Either do this locally but don't push, or just pass `DERA_GP_SIM_TESTS=ON`. See [here](http://stackoverflow.com/questions/18435516/how-to-set-a-cmake-option-at-command-line)